.azure-devops/templates/trigger-tests.yml

```diff
@@ -121,6 +121,21 @@ jobs:
       pythonVersion: $(python)
```

Comment:
Reminder to add user facing changes in HISTORY.

Reply:
have history in the pr description

Reply:
And you are writing automation to add it to our HISTORY/changelog file upon merge?

Reply:
that will be a future task

for now, I will keep this convo open till I merge it in